src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,43 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] result = new int[matrix.length];
+        for (int i = 0;i < matrix.length;i++) {
```

Comment:
```suggestion
        for (int i = 0; i < matrix.length; i++) {
```
don't forget about checkstyle

Reply:
checkstyle plugin said its fine, but fixed anyway

Reply:
yeah, checkstyle could not catch this option. Unfortunately